Fixed table sort in NewRelicFetchComponent (#19771)
backstage/backstage: .changeset/mean-queens-act.md

---
'@backstage/plugin-newrelic': patch
---

> Fixed bug in NewRelicComponent component where table would not sort correctly for numerical values.

Reviewer: ```suggestion
Fix sorting bug in `NewRelicFetchComponent` table by making all values numerical.
```

Author: Thank you for the fast review! Look forward to contributing more 🎉
